includes/definitions/zxr10.yaml

```diff
@@ -3,6 +3,11 @@ group: zxr10
 text: 'ZTE ZXR10'
 type: network
 icon: zte
+mib_dir: zte
```

Comment:
These are the defaults, you don't need to set them.

Reply:
I tried remove it and catched mib-not-found errors while run discovery.php

MIB search path: /opt/librenms/mibs
Cannot find module (ZXR10-MIB): At line 9 in (none)
zxr10SystemCpuUtility5m: Unknown Object Identifier (Sub-id not found: (top) -> zxr10SystemCpuUtility5m)
 
MIB search path: /opt/librenms/mibs
Cannot find module (ZXR10-MIB): At line 9 in (none)
ZXR10-MIB::zxr10SystemMemSize: Unknown Object Identifier 

So it must be there